### PR TITLE
IPsec documentation updated.

### PR DESCRIPTION
**Pull request by viktor-kurchenko**
IPsec documentation updated with new Cilium CLI commands.

### CODE REVIEW

File: Documentation/security/network/encryption-ipsec.rst [hunk 1]
```diff
@@ -162,7 +162,13 @@ commands:
 Key Rotation
 ============
 
-To replace cilium-ipsec-keys secret with a new key:
+Starting from Cilium CLI v0.15.23 IPsec keys can be rotated using the following command:
```
Comment (qmonnet) [inline Documentation/security/network/encryption-ipsec.rst]:
```suggestion
Starting from Cilium CLI v0.15.23, you can rotate the IPsec keys using the following command:
```

File: Documentation/security/network/encryption-ipsec.rst [hunk 2]
```diff
@@ -162,7 +162,13 @@ commands:
...
+
+.. code-block:: shell-session
+
+    cilium encryption rotate-key
+
+Alternatively, the following commands can be used to obtain the same result:
```
Comment (qmonnet) [inline Documentation/security/network/encryption-ipsec.rst]:
```suggestion
Alternatively, you can use the following commands to obtain the same result:
```

File: Documentation/security/network/encryption-ipsec.rst [hunk 3]
```diff
@@ -187,6 +193,16 @@ to all clusters in the mesh. You might need to increase the transition time to
 allow for the new keys to be deployed and applied across all clusters,
 which you can do with the agent flag ``ipsec-key-rotation-duration``.
 
+Encryption status
+=================
+
+Starting from Cilium CLI v0.15.23 cluster wide encryption status and IPsec key can be displayed using the following commands:
```
Comment (qmonnet) [inline Documentation/security/network/encryption-ipsec.rst]: This read like the key itself can be displayed, is it the case?

```suggestion
Starting from Cilium CLI v0.15.23, you can display cluster-wide encryption status and the status of the IPsec key with the following commands:
```

**Review by qmonnet — CHANGES_REQUESTED** [review]
Thanks! Looks good, with a few minor nits

**Comment by qmonnet** [comment]
This probably applies to stable versions too, so we probably want to backport the change?

File: Documentation/security/network/encryption-ipsec.rst (hunk 1 above)
Comment (viktor-kurchenko) [reply Documentation/security/network/encryption-ipsec.rst]: Fixed, thanks!

File: Documentation/security/network/encryption-ipsec.rst (hunk 2 above)
Comment (viktor-kurchenko) [reply Documentation/security/network/encryption-ipsec.rst]: Fixed, thanks!

File: Documentation/security/network/encryption-ipsec.rst (hunk 3 above)
Comment (viktor-kurchenko) [reply Documentation/security/network/encryption-ipsec.rst]: Fixed, thanks!

**Comment by viktor-kurchenko** [comment]
> This probably applies to stable versions too, so I suppose we want to backport the change?

Yes, I've added backport labels. Thanks!

**Review by qmonnet — APPROVED** [review]
Thank you!

**Comment by viktor-kurchenko** [comment]
@pchaigno gentle ping)

**Review by pchaigno — CHANGES_REQUESTED** [review]
I think we need to have this new command fully covered in CI before we can tell users to use it for rotations.

**Comment by qmonnet** [comment]
Is this PR still relevant today?

**Pull request closed without merging**